### DISCUSSION TITLE
幻影坦克

### DISCUSSION THREAD

**Discussion by guohuiyuan** [FloatTech/ZeroBot-Plugin#207] [Ideas]
功能：合成两个图片，防屏蔽，搞个类似https://github.com/kosakarin/hide_img的插件；

**Comment by fumiama** [comment]
但是问题在于被屏蔽的图在一开始就发不出来😂

**Reply by guohuiyuan** [reply]
说不定改个图片的属性就能发出来呢，新图片是合成的，玄学
